code/go/pkg/validator/validator.go

```diff
@@ -10,14 +10,31 @@ import (
 	"fmt"
 	"io/fs"
 	"os"
+	"path/filepath"
 
 	"github.com/elastic/package-spec/v3/code/go/internal/packages"
 	"github.com/elastic/package-spec/v3/code/go/internal/validator"
+	"github.com/elastic/package-spec/v3/code/go/pkg/linkedfiles"
 )
 
 // ValidateFromPath validates a package located at the given path against the
 // appropriate specification and returns any errors.
 func ValidateFromPath(packageRootPath string) error {
+	root, err := linkedfiles.FindRepositoryRoot()
+	if err != nil {
+		return err
+	}
```

Comment:
This function already receives the root of the package, we should be able to rely on it.
```suggestion
	root, err := os.OpenRoot(packageRootPath)
	if err != nil {
		return nil, fmt.Errorf("could not open root: %w", err)
	}
```